cookbooks/aws-parallelcluster-platform/resources/gdrcopy/gdrcopy_amazon2.rb

```diff
@@ -25,6 +25,14 @@ def gdrcopy_platform
   'amzn-2'
 end
 
+def gdrcopy_build_dependencies
+  if aws_region.start_with?("us-iso")
+    %w(dkms rpm-build make check check-devel)
+  else
+    %w(dkms rpm-build make check check-devel subunit subunit-devel)
```

Comment:
Why some is removed?

Reply:
those are not available in ADC regions, so they are downloaded in install_packages through the `epel_deps.tar.gz` artifiact.